x11-misc/xfe/xfe-1.44.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PLOCALES="
+	bs ca cs da de el es_AR es_CO es fr hu it ja nl no pl pt_BR pt_PT ru sv tr
+	zh_CN zh_TW
+"
+inherit plocale xdg-utils
+
+DESCRIPTION="MS-Explorer-like minimalist file manager for X"
+HOMEPAGE="http://roland65.free.fr/xfe"
+SRC_URI="mirror://sourceforge/${PN}/${PN}/${PV}/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="debug nls startup-notification"
+
+RDEPEND="
+	media-libs/libpng:0
+	x11-libs/fox:1.6[png,truetype]
+	x11-libs/libX11
+	x11-libs/libXft
+	startup-notification? ( x11-libs/startup-notification )
+"
```

Comment:
Misses

```
media-libs/fontconfig
x11-libs/libXrandr
x11-libs/libxcb:=
x11-libs/xcb-util
```

Reply:
thanks for the tipp :D when I use qa-vdb It just tells me about media-libs/fontconfig and x11-libs/libXrandr. Did you use something else?

Reply:
Some of the deps only appear with specific USE flags enabled.